scipy/linalg/flapack_gen_tri.pyf.src

```diff
@@ -13,3 +13,54 @@ subroutine <prefix>gtsv(n, nrhs, dl, d, du, b, info)
     integer intent(out) :: info
 
 end subroutine <prefix>gtsv
+
+
+subroutine <prefix>gttrf(n, dl, d, du, du2, ipiv, info)
+    ! dl, d, du, du2, ipiv, info  = gttrf(dl, d, du, [overwrite_dl=0, overwrite_d=0, overwrite_du=0])
+    ! ?GTTRF computes an LU factorization of a tridiagonal matrix A
+    ! using elimination with partial pivoting and row interchanges.
+    !
+    ! The factorization has the form
+    !    A = L * U
+    !
+    ! where L is a product of permutation and unit lower bidiagonal
+    ! matrices and U is upper triangular with nonzeros in only the main
+    ! diagonal and first two superdiagonals.
+    callstatement (*f2py_func)(&n, dl, d, du, du2, ipiv, &info)
+    callprotoargument int*, <ctype>*, <ctype>*, <ctype>*, <ctype>*, int*, int*
+
+    integer intent(hide),   depend(d), check(len(d) > 2) :: n = len(d)
+    <ftype> intent(in,out,copy), depend(n), dimension(n-1) :: dl
```

Comment:
At the output these need new names so add also `in, out, copy, out=varname` as the intent

Reply:
do you have any suggestions for names? Maybe `m` for multiples?

Reply:
I agree that `b` (below) should be `x` on output for `?gttrs`, but do these really need different names?  [`?gtsv`](https://17626-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.linalg.lapack.dgtsv.html#scipy.linalg.lapack.dgtsv) didn't rename them.

If they must, I'd suggest appending `_o` suffix.

Reply:
Left these alone, fixed `b` -> `x` below.